lib/index.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+const nunjucks = require('nunjucks');
+
+nunjucks.configure('../templates', { autoescape: false });
+
+
+const NGINX_TEMPLATE_FILE = 'nginx.conf.nj';
+
+function getPageConfigs(configFile) {
+  const contents = yaml.safeLoad(fs.readFileSync(configFile, 'utf-8'));
+  const configs = contents.map((c) => {
+    if (typeof c === 'string') {
+      return { from: c, to: c };
+    }
+    if (!c.from || !c.to) {
+      throw new Error('Non-string configs must have "to" and "from" properties');
+    }
+    return c;
+  });
+  return configs;
```

Comment:
I noticed these are plopped right into the regexps in the generated `nginx.conf` file without any escaping, etc.  Do you think this should be documented, or do you think the `from` and `to` keys should be validated to e.g. ensure that they conform to a fairly restricted set of characters?